### PR TITLE
Adding blog post about tokenized memberships

### PR DESCRIPTION
**Pull request by julien51**
# Description

All in the title.

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by julien51** [comment]
image

File: unlock-protocol.com/blog/tokenizing-memberships.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+subTitle: What if there was a new way to interact with the communities you are part of?
+authorName: Julien Genestoux
+publishDate: October 18, 2019
+description: What if there was a new way to interact with the communities you are part of?
+image: /static/images/blog/tokenized-memberships.jpg
+---
+
+The web needs a [new business model](https://medium.com/unlock-protocol/its-time-to-unlock-the-web-b98e9b94add1) — one which doesn’t erode people’s privacy or centralize control into the hands of a few powerful entities.
+
+Luckily, the web is resilient, it’s reliable, and it’s really, really good at lots of things. Here’s one example: it’s great at creating communities.
```
Comment (cnasc) [inline unlock-protocol.com/blog/tokenizing-memberships.md]:
```suggestion
Luckily, the web is resilient, reliable, and really, really good at lots of things. Here’s one example: it’s great at creating communities.
```

Removing the "it's"s really gets that alliteration going. Also we may want a better way to say "good at lots of things."

File: unlock-protocol.com/blog/tokenizing-memberships.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+The keys themselves can have multiple purposes: the fact that you follow Unlock’s Github account could mean you automatically get invited to our launch party. And, if it’s a token, it doesn’t matter whether Github decides to suddenly remove its ⭐️ feature: tokens mean you would still be a member of our loyal developer community.
+
+You could also use these tokens to transfer memberships to somebody else, or even to burn the membership itself.
+
+# It’s not (just) about the money
+
+Memberships are powerful things. They can be about social status, showing support, or just belonging. And they can include financial elements too. Many people proudly show their love for a brand by showing the logos on their clothes, their cars, their laptop. Non-profits realized a long time ago that getting somebody to wear a colored ribbon 🎗was sometimes as powerful as getting a donation from them. And sports teams are already experts at monetizing their relationships with members by selling hats, jerseys, and gear with the club colors.
```
Comment (cnasc) [inline unlock-protocol.com/blog/tokenizing-memberships.md]:
```suggestion
Memberships are powerful things. They can be about social status, showing support, or just belonging. And they can include financial elements too. Many people proudly show their love for a brand by showing the logos on their clothes, their cars, their laptop. Non-profits realized a long time ago that getting somebody to wear a colored ribbon 🎗 was sometimes as powerful as getting a donation from them. And sports teams are already experts at monetizing their relationships with members by selling hats, jerseys, and gear with the club colors.
```

Comment (cnasc) [reply unlock-protocol.com/blog/tokenizing-memberships.md]: (adding a space between the ribbon emoji and the word "was")

**Pull request merged**